mono/tests/sgen-toggleref.cs

```diff
@@ -19,17 +19,45 @@ public class Toggleref {
 	}
```

Comment:
So if this test is not run with netcore, it doesn't need to be modified.

Reply:
We want to run the tests on netcore at some point so might still be worth doing it.

Reply:
Okay! I will redo!